packages/angular-workspace/nimble-angular/src/directives/list-option-group/nimble-list-option-group.directive.ts

```diff
@@ -3,7 +3,7 @@ import { type BooleanValueOrAttribute, toBooleanProperty } from '@ni/nimble-angu
 import type { ListOptionGroup, listOptionGroupTag } from '@ni/nimble-components/dist/esm/list-option-group';
 
 export type { ListOptionGroup };
-export { listOptionGroupTag };
```

Comment:
Any idea why this tag is exported with `type` when it doesn't look like many of the other component tags are?

Reply:
Well, the difference between this (and `NimbleListOptionDirective`, which also has this change), is that they are importing the members differently. These two both import the tag with the `type` keyword, where other directives do not. This still seems like a mistake in that the tag being exported with as a type seems wrong.

Reply:
Great catch. I'll fix them up.